stack_validation_test.go

```diff
@@ -0,0 +1,82 @@
+package lifecycle_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/buildpacks/imgutil/fakes"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/buildpacks/lifecycle"
+	"github.com/buildpacks/lifecycle/platform"
+	h "github.com/buildpacks/lifecycle/testhelpers"
+)
+
+func TestStackValidation(t *testing.T) {
+	spec.Run(t, "StackValidation", testStackValidation, spec.Report(report.Terminal{}))
+}
+
+func testStackValidation(t *testing.T, when spec.G, it spec.S) {
+	when(".ValidateStack", func() {
```

Comment:
```suggestion
	when("ValidateStack", func() {
```

Reply:
I saw that we're adding this `.` sometimes (for example in `utils_test.go`), but I'm not sure why we're doing it...